weasyl/templates/help/community.html

```diff
@@ -22,20 +22,29 @@ <h4>A. Right to Submit</h4>
         <li><strong>Copyright and Trademark:</strong> The creator of the content of a submission holds the copyright and rights to its distribution; unless the content is trademarked by someone else, is considered harassment, or is otherwise a violation of Weasyl policy, the submission will not be removed. For further information, see our <a href="/policy/copyright">Copyright Policy</a>.</li>
       </ol>
 
-      <h4>B. Accurately Describing Content</h4>
+      <h4>B. Rating and Describing Your Content</h4>
+
+      <ul>
+        <li><strong>Terminology</strong>
+          <p>A “Submission” is any artwork, story, comic page, still image, or animation that the user uploads to the site. “Content” refers to actions, poses, points of view or mise-en-scene involving characters (see below) in the submission.</p>
+          <p>A “Character” is a drawn or described depiction of a conscious being (e.g., a furry, an anthropomorphized entity, an animal, a human, a robot, or any combination of these) in the submission. A “Cub” (sometimes called a “Babyfur” or “kidfur”) is any character with recognizably humanoid body proportions of children or infants (see below). We are using the term “cub” here as most characters on Weasyl are anthropomorphic animals. NOTE: Not to be confused with a “minor” (i.e., a user of the site who is under the real-life age of 18 and restricted to viewing General content).</p>
+          <p>A “Rating”: All submissions must, at the time of upload, be assigned one of three ratings: General, Mature, or Explicit; content must be rated appropriately.</p>
+          <p>What “Sexual Content” Is: Submissions may include sexually explicit content, either implied or explicit, if rated appropriately. For the purposes of this document, “sexually explicit content” is anything involving but not necessarily limited to: exposed or aroused genitalia, including genital bulges/”tents” or partial concealment of aroused genitals, frotting, masturbation with one character or any form of sexual intercourse between two or more characters, or the depiction or use of sex toys. Furthermore, characters in provocative “pinup” poses who are presenting their genital or perianal regions in a provocative manner can be sexual content, even if those genital regions are concealed. Hyper-enlarged genitals or breasts in any context are sexual content. Certain fetishes and kinks such as BDSM qualify as sexual or explicit content. <em>The final decision of what constitutes “sexual content” or what is merely suggestive is determined by Weasyl’s moderators.</em></p>
```

Comment:
> Furthermore, characters in provocative “pinup” poses who are presenting their genital or perianal regions in a provocative manner can be sexual content, even if those genital regions are concealed.

This may be original wording but opens up a grey area or a debating point with users about how the rules are meant to be interpreted. I'd rather this be clear so users can be sure of where they stand (and if we end up with a bad mod on team one day, they can't use the ambiguity as a stick to beat users with)